book/src/jsonrpc-api.md

```diff
@@ -498,6 +474,33 @@ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":1, "m
 
 ---
 
+### getVoteAccounts
+Returns the account info and associated stake for all the voting accounts in the current bank.
+
+##### Parameters:
+None
+
+##### Results:
+The result field will be a JSON object of `current` and `delinquent` accounts,
+each containing an array of JSON objects with the following sub fields:
+* `votePubkey` - Vote account public key, as base-58 encoded string
+* `nodePubkey` - Node public key, as base-58 encoded string
+* `stake` - the stake, in lamports, delegated to this vote account
```

Comment:
This is actually "activatedStake" right?

Reply:
Yes, I believe you're right, if I'm understanding `Stake::stake()` correctly.
Perhaps we should upgrade from the anonymous `u64` in runtime: https://github.com/solana-labs/solana/blob/b28407d98a44919a78916d1f8ce85cbdd2972996/runtime/src/stakes.rs#L14